renderer/nyaovim-app.ts

```diff
@@ -209,7 +209,7 @@ function prepareIpc(client: Nvim) {
             } else if (ch === 'i') {
                 // insert mode
                 // gp will move cursor to the last of pasted content
-                command = '<C-o>"+gp';
+                command = '"+gp';
```

Comment:
Looks `<C-o>` is removed. Does this work in inert mode? (I only check this diff in GitHub. So if you specify `^o` directly, I may miss it...)